docs/docs/50-user-guide/70-annotations.md

```diff
@@ -0,0 +1,74 @@
+---
+sidebar_label: Annotations
+description: List of Kargo-specific annotations and labels
+---
+
+# Annotations and Labels used by Kargo
+
+This page documents Kargo-specific annotations, labels, and finalizers. Some of these fields can be set by users, while others are system-managed and used internally by the control plane.
+
+## Annotations
+
+| Annotation key | Target resource(s) | Possible values | Description |
+|----------------|--------------------|------------------|-------------|
+| `kargo.akuity.io/refresh` | Any Kargo resource | Arbitrary string (acts as a token) | Triggers reconciliation when the value changes. Useful for forcing a resync. |
+| `kargo.akuity.io/reverify` | `Stage` | Verification ID or JSON object (VerificationRequest) | Triggers re-verification of the currently assigned Freight. |
+| `kargo.akuity.io/abort` | `Stage` | Verification ID | Aborts an in-progress Freight verification. |
+| `kargo.akuity.io/description` | Any | Free-form string | Human-readable description displayed in the UI. |
```

Comment:
I'm not sure that the UI actually honors this for _any_ resource type as has been indicated here.